### ISSUE TITLE
Remove certain scan scripts for the initial release 

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#453]
* 1.4.3 Contrast (Minimum) [AA]

   * Technique G18
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of {value}. 
Recommendation: {colour recommendations}. 
[Standard].Principle1.Guideline1_4.1_4_3.G18

   * Technique G145
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of {value}. 
Recommendation: {colour recommendations}. 
[Standard].Principle1.Guideline1_4.1_4_3.G145

* 3.1.1 Language of Page [A]

   * Technique H57

The html element should have a lang or xml:lang attribute which describes the language of the document.
[Standard].Principle3.Guideline3_1.3_1_1.H57.2
The language specified in the lang attribute of the document element does not appear to be well-formed. 
[Standard].Principle3.Guideline3_1.3_1_1.H57.3.Lang
The language specified in the xml:lang attribute of the document element does not appear to be well-formed. 
[Standard].Principle3.Guideline3_1.3_1_1.H57.3.XmlLang

* 3.1.2 Language of Parts [AA]

   * Technique H58
The language specified in the lang attribute of this element does not appear to be well-formed. 
[Standard].Principle3.Guideline3_1.3_1_2.H58.1.Lang
The language specified in the xml:lang attribute of this element does not appear to be well-formed. 
[Standard].Principle3.Guideline3_1.3_1_2.H58.1.XmlLang

* 4.1.1 Parsing [A]

   * Technique F77
Duplicate id attribute value "[Element ID]" found on the web page.
[Standard].Principle4.Guideline4_1.4_1_1.F77

**Comment by gbinal** [comment]
This is done and in place in scans now.

**Comment by gbinal** [comment]
Specifically, here's the filter we are using:  

````
'ignore': [
                'notice',
                'warning',
                'WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.BgImage',
                'WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Abs',
                'WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Abs',
                'WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2',
                'WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.3',
                'WCAG2AA.Principle3.Guideline3_1.3_1_2.H58.1',
                'WCAG2AA.Principle4.Guideline4_1.4_1_1.F77'
            ]
````
